packages/core/src/html/NodeProcessor.ts

```diff
@@ -291,12 +327,20 @@ class NodeProcessor {
     case 'md':
       node.name = 'span';
       node.children = cheerio.parseHTML(
-        this.markdownProcessor.renderMdInline(cheerio.html(node.children)), true);
+        // TODO figure this typing out
+        this.markdownProcessor.renderMdInline(cheerio.html(node.children as unknown as cheerio.Element)),
+        undefined, true,
+        // TODO figure this typing out
+      ) as unknown as DomElement[];
       break;
     case 'markdown':
       node.name = 'div';
       node.children = cheerio.parseHTML(
-        this.markdownProcessor.renderMd(cheerio.html(node.children)), true);
+        // TODO figure this typing out
+        this.markdownProcessor.renderMd(cheerio.html(node.children as unknown as cheerio.Element)),
+        undefined, true,
+        // TODO figure this typing out
```

Comment:
a more horrendous typing issue with cheerio/htmlparser2. Functionally the same but dosen't look pretty.